writer.go

```diff
@@ -76,3 +75,12 @@ func open(paths []string) ([]zapcore.WriteSyncer, func(), error) {
 	}
 	return writers, close, errs.AsError()
 }
+
+// OpenWriteSyncers combines the passed set of WriteSyncer objects into a locked
+// WriteSyncer. It returns any error encountered.
+func OpenWriteSyncers(writers ...zapcore.WriteSyncer) (zapcore.WriteSyncer, error) {
+	if len(writers) == 0 {
+		return zapcore.AddSync(ioutil.Discard), nil
+	}
+	return zapcore.Lock(zapcore.NewMultiWriteSyncer(writers...)), nil
+}
```

Comment:
Ack, my fault - I confused this with the previously-present `OpenWriters`.

Since the files/writers are already open, let's rename this to `CombineWriteSyncers` or `ComposeWriteSyncers` instead. This function can't fail, so let's also remove the error return value.